### ISSUE TITLE
Mailchimp - Subscriber Pop-up

### ISSUE THREAD

**Issue by jshauffer** [j26design/Mailchimp#5]
Have you made or are making a Subscriber Pop-up widget for Mailchimp to work with Adobe Muse?

**Comment by jp26jp** [comment]
Is there any particular benefit to using Mailchimp's version compared to using the popup overlay that muse offers?

**Comment by jshauffer** [comment]
No...you are right.

What my client wanted was for the form to pop up after 8-9 seconds after
each page loads.

Your Mailchimp widgets are amazing but I had no option for the auto
popup...and that was the original reason for my email.

I could use the built-in Adobe Muse lightbox widget which can now be turned
on to start but there is no timer option for that.

I found a QooQee widget that I combined with you mailchimp widget.

Here is a link to the final website: bestliferesults.com

Keep up the great work.

Thank you!





On Fri, Feb 10, 2017 at 8:17 AM John R Perry <notifications@github.com>
wrote:

Is there any particular benefit to using Mailchimp's version compared to
using the popup overlay that muse offers?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/j26design/Mailchimp/issues/5#issuecomment-278939738>,
or mute the thread
